pandas/io/tests/test_pickle.py

```diff
@@ -86,6 +86,13 @@ def compare(self, vf, version):
                 comparator(result, expected, typ, version)
         return data
 
+    def compare_sp_series_ts(self, res, exp, typ, version):
+        # SparseTimeSeries deprecated in 0.17.0
+        if version and LooseVersion(version) < "0.17.0":
```

Comment:
Thx for the update. I think `` ... <= "0.12.0"`` is OK based on the error below. After that, ``SparseTimeSeries`` has been integrated to ``SparseSeries``.

- https://travis-ci.org/pydata/pandas/jobs/147445144